hoot-core/src/main/cpp/hoot/core/conflate/highway/ImpliedDividedMarker.cpp

```diff
@@ -125,6 +128,7 @@ boost::shared_ptr<OsmMap> ImpliedDividedMarker::markDivided()
     {
       // mark this tunnel/bridge as divided.
       w->setTag("divider", "yes");
+      _numMarked++;
     }
```

Comment:
Added a bunch of counters like this to ops and visitors (only a subset of the ones involved in cleaning for now) to help better see how much cleaning is affecting the input data.